spec/dummy/db/migrate/20150713035548_add_function_count_pets.rb

```diff
@@ -0,0 +1,22 @@
+class AddFunctionCountPets < ActiveRecord::Migration
+  def change
+    create_function 'pets_not_empty()', :boolean, <<-FUNCTION, schema: 'public'
+BEGIN
+  IF (SELECT COUNT(*) FROM pets) > 0
+  THEN
+    RETURN true;
+  ELSE
+    RETURN false;
+  END IF;
+END;
```

Comment:
Here's my problem. I see this regex and the first thing I ask is why is it here and what useful function does it serve? Thoroughly not convinced the extra cognitive load makes for a good tradeoff.